Gemfile

```diff
@@ -1,10 +1,15 @@
-# If you have OpenSSL installed, we recommend updating
-# the following line to use "https"
-source "http://rubygems.org"
+# If you do not have OpenSSL installed, change
+# the following line to use "http://"
+source "https://rubygems.org"
+
+# Uncomment these gems if you are using Windows
+# gem "wdm", "~> 0.1.0", platforms: [:mswin, :mingw]
+# gem "tzinfo-data", platforms: [:mswin, :mingw, :jruby]
 
 gem "middleman", ">= 4.0.0"
```

Comment:
Could we specify the version numbers for some of the gems here? For example middleman would use the release candidate version `4.1.0.rc.1` with the current `>= 4.0.0` specification in the gemfile.

Also how do we feel about the gemfile being in alphabetical order? 
```
gem "bourbon", "~> 4.2"
gem "middleman", "~> 4.0"
gem "middleman-autoprefixer", "~> 2.6"
gem "middleman-deploy", "= 2.0.0.pre.alpha"
gem "middleman-livereload", "~> 3.4"
gem "middleman-sprockets", "= 4.0.0.rc.1"
gem "neat", "~> 1.7"
```